src/Uno.UI/MixinGeneration.targets

```diff
@@ -29,11 +29,6 @@
 	</ProjectReference>
   </ItemGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.BannedApiAnalyzers" Version="3.3.2" />
-    <AdditionalFiles Include="$(MSBuildThisFileDirectory)\BannedSymbols.txt" />
-  </ItemGroup>
-
```

Comment:
Moved this to Directory.Build.targets since Mixins are expected to be removed in the future. So this is to prevent unintentional regression.